pinot-core/src/main/java/org/apache/pinot/core/auth/Actions.java

```diff
@@ -89,6 +89,7 @@ public static class Cluster {
 
   // Action names for table
   public static class Table {
+    public static final String ABORT_REBALANCE = "AbortRebalance";
```

Comment:
We might want to call it `CancelRebalance` (we have `CancelQuery`)

Reply:
sgtm, particularly with the CANCELLED status.